core/src/main/scala/org/apache/spark/scheduler/cluster/SimrSchedulerBackend.scala

```diff
@@ -38,11 +39,9 @@ private[spark] class SimrSchedulerBackend(
   override def start() {
     super.start()
 
-    val driverUrl = "akka.tcp://%s@%s:%s/user/%s".format(
-      SparkEnv.driverActorSystemName,
-      sc.conf.get("spark.driver.host"),
-      sc.conf.get("spark.driver.port"),
-      CoarseGrainedSchedulerBackend.ACTOR_NAME)
+    val driverUrl = AkkaUtils.address(SparkEnv.driverActorSystemName,
```

Comment:
Minor readability nit, but I think this would be a little easier to parse if you split it with one argument per line, as you've done in SparkDeploySchedulerBackend.